recipes/highway/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class HighwayConan(ConanFile):
+    name = "highway"
+    description = "Performance-portable, length-agnostic SIMD with runtime " \
+                  "dispatch"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/highway"
+    topics = ("highway", "simd")
+
+    settings = "os", "compiler", "build_type", "arch"
+
+    exports_sources = "CMakeLists.txt", "patches/**"
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 11
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "15",
+            "gcc": "8",
+            "clang": "7",
+        }
```

Comment:
They claim to support older compilers. Can't you build hwy target only?

Reply:
Conan build bot failed on the older compilers (see https://github.com/conan-io/conan-center-index/pull/4996#issuecomment-803603675). And there were errors in the core files, so it won't help if I remove everything else.

Also, they mention testing on specific versions of compilers. On these versions the build succeeds.

>periodically tested on x86 with Clang 7-11 and GCC 8, 9 and 10.2.1